src/main/java/org/apache/sysml/api/DMLScript.java

```diff
@@ -188,15 +189,17 @@ public static void main(String[] args)
 		String[] otherArgs = new GenericOptionsParser(conf, args).getRemainingArgs();
 		
 		try {
-		DMLScript.executeScript(conf, otherArgs); 
+			DMLScript.executeScript(conf, otherArgs);
+		} catch (ParseException pe) {
+			System.err.println(pe.getMessage());
 		} catch (DMLScriptException e){
 			// In case of DMLScriptException, simply print the error message.
 			System.err.println(e.getMessage());
 		}
 	} 
 
 	public static boolean executeScript( String[] args ) 
-		throws DMLException
+		throws DMLException, ParseException
```

Comment:
If you're going to expose ParseException here, consider either renaming it to DMLParseException or making it a subclass of DMLException. Other systems and libraries use the name "ParseException", and we don't want users to have to type enter fully-qualified class names because they ended up with code that throws two kinds of "ParseException".